src/Input/index.js

```diff
@@ -1,7 +1,20 @@
-import React from "react";
+import React, {useState} from "react";
 
 const Input = () => {
-  return <input />;
+  const [text, setTxt] = useState('');
+  const displayNull = '값을 입력해주세요';
```

Comment:
👍 
```suggestion
  const DISPLAY_EMPTY = '값을 입력해주세요';
```
상수 개념이니깐 대문자로 하면 좋을 것 같아요